Geometry/EcalCommonData/interface/DDEcalEndcapTrapX.h

```diff
@@ -0,0 +1,53 @@
+#ifndef Geometry_EcalCommonDatao_DDEcalEndcapTrapX_h
+#define Geometry_EcalCommonDatao_DDEcalEndcapTrapX_h
+
+#include <vector>
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DetectorDescription/DDCMS/interface/DDRotationMatrix.h"
+#include "DetectorDescription/DDCMS/interface/DDTranslation.h"
+
+// Define Endcap Supercrystal class
+
```

Comment:
It would be good to add a comment here saying that this file is the DD4hep version that should become the only version when DDD is eliminated.

Reply:
> It would be good to add a comment here saying that this file is the DD4hep version that should become the only version when DDD is eliminated.

@bsunanda Would you please add the comment?

Reply:
@qliphy @silviodonato @cvuosalo  added the comment in PR description